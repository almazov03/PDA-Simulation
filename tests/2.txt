STARTING_NON_TERMINAL=$S$
$S$ = {$X$ $A$}
$S$ = {$B$ $B$}
$B$ = {"b"}
$B$ = { $S$ $B$ }
$X$ = { "b" }
$A$ = { "a" }