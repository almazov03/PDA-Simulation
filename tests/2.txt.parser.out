STARTING_NON_TERMINAL=$S$
$B$ = { "b" }
$X$ = { "b" }
$A$ = { "a" }
$B$ = { "b" $B'$ }
$B'$ = { "b" $B$ $B'$ }
$B'$ = { "b" $B'$ $B$ $B'$ }
$B'$ = { "b" $B$ }
$B'$ = { "b" $B'$ $B$ }
$B'$ = { "b" $A$ $B$ $B$ $B'$ }
$B'$ = { "b" $A$ $B$ $B'$ $B$ $B'$ }
$B'$ = { "b" $A$ $B$ $B$ }
$B'$ = { "b" $A$ $B$ $B'$ $B$ }
$S$ = { "b" $B$ }
$S$ = { "b" $B'$ $B$ }
$S$ = { "b" $A$ }
$S$ = { "b" $A$ $B$ $B$ }
$S$ = { "b" $A$ $B$ $B'$ $B$ }
$B$ = { "b" $A$ $B$ }
$B$ = { "b" $A$ $B$ $B'$ }

